pkgs/development/libraries/duckdb/default.nix

```
@@ -97,10 +97,13 @@ stdenv.mkDerivation (finalAttrs: {
         "test/sql/attach/attach_remote.test"
         "test/sql/copy/csv/test_sniff_httpfs.test"
         "test/sql/httpfs/internal_issue_2490.test"
+        # fails with incorrect result
```

```suggestion
        # fails with incorrect result
        # Upstream issue https://github.com/duckdb/duckdb/issues/14294
```

I've opened issue upstream. I think we should add the link for future reference.

Don't forget to squash commits after applying suggestion. Once we have a link (and feel free to use different working if you wish) It will be ready for merge from my side.